CHANGELOG-3.2.md

```diff
@@ -28,7 +28,7 @@ Note that any `etcd_debugging_*` metrics are experimental and subject to change.
 - Add [`etcd_mvcc_db_total_size_in_bytes`](https://github.com/coreos/etcd/pull/9819) Prometheus metric.
   - In addition to [`etcd_debugging_mvcc_db_total_size_in_bytes`](https://github.com/coreos/etcd/pull/9819).
 - Add [`etcd_mvcc_db_total_size_in_use_in_bytes`](https://github.com/coreos/etcd/pull/9256) Prometheus metric.
-  - Use it with `etcd_mvcc_db_total_size_in_bytes` and `etcd_mvcc_db_total_size_in_use_in_bytes`.
+  - Use it with `etcd_server_quota_backend_bytes` and `etcd_mvcc_db_total_size_in_use_in_bytes`.
```

Comment:
revert this change? We already have this change log in line 22?

Reply:
It is kinda repeating with line 22. We could either remove line 31 completely, or I think we were trying to say use the three metrics (etcd_mvcc_db_total_size_in_use_in_bytes, etcd_server_quota_backend_bytes and etcd_mvcc_db_total_size_in_bytes) together, before it mentioned etcd_mvcc_db_total_size_in_use_in_bytes again but didn't say etcd_server_quota_backend_bytes. Well, apparently I changed the wrong thing though, I meant to s/etcd_mvcc_db_total_size_in_use_in_bytes/etcd_server_quota_backend_bytes

Reply:
Oh, I just noticed that. Yeah they are redundant.